ghost/core/core/frontend/src/member-attribution/member-attribution.js

```diff
@@ -71,6 +71,26 @@ const LIMIT = 15;
             history = [];
         }
 
+        // Fetch referrer URL from document
+        const referrerUrl = window.document.referrer ? new URL(window.document.referrer).hostname : null;
+        let refParam;
+        let sourceParam;
+        let utmSourceParam;
+        let utmMediumParam;
+        try {
+            // Fetch source/medium from query param
+            const url = new URL(window.location.href);
+            refParam = url.searchParams.get('ref');
+            sourceParam = url.searchParams.get('source');
+            utmSourceParam = url.searchParams.get('utm_source');
+            utmMediumParam = url.searchParams.get('utm_medium');
```

Comment:
Should we remove these params from the URL afterwards?

Reply:
🤔 I was thinking about it too, specially after looking at the fact we remove it for attribution id/sources, but i think these params are different because they are set externally, and there might be other analytics/tools that are setup to rely on them, so imo doesn't make sense for us to remove them, wdyt ?

Reply:
You are right about external tools, didn't think about that 🤔

Reply:
Will keep it in for now, and based on how it feels later on testing we can remove it if needed